endpoints/errors.go

```diff
@@ -69,39 +55,39 @@ func NewAPIError(name string, msg string, code int) error {
 	return &APIError{Name: name, Msg: msg, Code: code}
 }
 
-// Errorf creates a new APIError for custom error given a format string and arguments.
-func Errorf(code int, name string, format string, args ...interface{}) error {
-	return &APIError{Name: name, Msg: fmt.Sprintf(format, args...), Code: code}
+// errorf creates a new APIError given its status code, a format string and its arguments.
+func errorf(code int, format string, args ...interface{}) error {
```

Comment:
So, you're basically saying if needs a custom error, they can just do `&APIError{...}`.